e2e/nomostest/registry_server.go

```diff
@@ -126,6 +126,35 @@ func (nt *NT) BuildAndPushOCIImage(repository *gitproviders.Repository) (*regist
 	return image, nil
 }
 
+// BuildAndPushHelmPackage uses the current file system state of the provided Repository
+// to build a helm package and push it to the current HelmProvider. The resulting
+// HelmPackage object can be used to set the spec.oci.image field on the RSync.
+func (nt *NT) BuildAndPushHelmPackage(repository *gitproviders.Repository) (*registryproviders.HelmPackage, error) {
```

Comment:
they're called charts, not packages in the helm universe.

Reply:
i guess they're packages in the context of config sync... just seems confusing to call them that. maybe it doesn't realy matter...

Reply:
I don't feel strongly, I'm fine with calling them either